docs/reference-architectures/serverless/web-app.md

```diff
@@ -115,15 +104,13 @@ By using bindings, you don't need to write code that talks directly to the servi
 
 **Cosmos DB**. Throughput capacity for Cosmos DB is measured in [Request Units][ru] (RU). A 1-RU throughput corresponds to the throughput need to GET a 1KB document. In order to scale a Cosmos DB container past 10,000 RU, you must specify a [partition key][partition-key] when you create the container and include the partition key in every document that you create. For more information about partition keys, see [Partition and scale in Azure Cosmos DB][cosmosdb-scale].
 
-**API Management**. API Management can scale out and supports rule-based autoscaling. Note that the scaling process takes at least 20 minutes. If your traffic is bursty, you should provision for the maximum burst traffic that you expect. However, autoscaling is useful for handling hourly or daily variations in traffic. For more information, see [Automatically scale an Azure API Management instance][apim-scale].
```

Comment:
Front Door doesn't have scaling considerations AFAIK because it's not regional. But I couldn't find good information on that to verify that's the case.

Also, the fact that APIM has to be scaled out means that it is not as serverless as Front Door. Part of the promise of serverless is the auto-scaling. We should try and stick to that story all the way through. I _think_ Front Door delivers on this promise.